wolfsentry/wolfssl_test.h

```diff
@@ -0,0 +1,576 @@
+/*
+ * wolfsentry.h
+ *
+ * Copyright (C) 2021-2023 wolfSSL Inc.
+ *
+ * This file is part of wolfSentry.
+ *
+ * wolfSentry is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSentry is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+/*! @file wolfssl_test.h
+    \brief Macros and helper functions for wolfSSL --enable-wolfsentry.
+
+    This file is included by wolfssl/test.h when defined(WOLFSSL_WOLFSENTRY_HOOKS).
+ */
+
+#include <wolfsentry/wolfsentry_util.h>
+
```

Comment:
please add a #ifndef guard to avoid multiple inclusions (or pragma once)

Reply:
:heavy_check_mark: